library/edihistory/edih_277_html.php

```
@@ -97,405 +97,369 @@ function edih_277_transaction_html($obj277, $bht03, $accordion = false)
+                case 'STC':
+                    //
+                    if (isset($sar[1])) {
+                        if (strpos($sar[1], (string) $ds)) {       // claim status category : claim status : entity identifier
+                            $scda = explode($ds, $sar[1]);
+                            $sc101 = ( isset($scda[0]) && $scda[0]) ? $cd27x->get_271_code('HCCSCC', $scda[0]) : "";
+                            $sc102 = ( isset($scda[1]) && $scda[1]) ? $cd27x->get_271_code('HCCSC', $scda[1]) : "";
+                            $sc103 = ( isset($scda[2]) && $scda[2]) ? $cd27x->get_271_code('NM101', $scda[2]) : "";
+                        }
                     } else {
-                        $stc03 = $sar[3];
+                        $stc01 = $sar[1];
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$stc01'. (undefined)

(UnusedLocalVariable)